pages/settings/FiatCurrency.tsx

```diff
@@ -111,7 +110,7 @@ function CurrencyList() {
                 {code}
               </Text>
             </TouchableOpacity>
-            {index === TOP_CURRENCIES.length - 1 && (
+            {priority !== 100 && filteredCurrencies[index + 1][2] === 100 && (
```

Comment:
This check is very fragile. I would not hard code 100 anywhere. Isn't it that you want to show the top priority items separately?

Reply:
Fixed